File: sprint1/Java/lesson5code1.java
int[] visitors = {0, 2, 3, 2, 0, 4, 1, 1, 2};
int[] entriesByVisitor = new int[5];
int bestVisitor = 0;

for (int i = 0; i < visitors.length; i++) {
    int visitor = visitors[i];
    entriesByVisitor[visitor]++;
    if (entriesByVisitor[visitor] > entriesByVisitor[bestVisitor]) {
        bestVisitor = visitor;
    }
}

System.out.println(bestVisitor);

File: sprint1/By lessons/lesson5code1/lesson5code1.kt
// visitors - массив номеров пассажиров.
// Каждый пассажир проехал столько раз, сколько раз встречается его номер
val visitors = intArrayOf(0, 2, 3, 2, 0, 4, 1, 1, 2)
val entriesByVisitor = IntArray(5)
var bestVisitor = 0

for (visitor in visitors) {
    entriesByVisitor[visitor] += 1
    if (entriesByVisitor[visitor] > entriesByVisitor[bestVisitor]) {
        bestVisitor = visitor
    }
}

println(bestVisitor)
